### PR TITLE
Add support for removing background images from smaller screens

### PR DESCRIPTION
**Pull request by aredridel**

### CODE REVIEW

**Comment by stubbornella** [comment]
Can you please provide a description and screenshot of what you made?

File: src/pivotal-ui/components/utils/utils.scss [hunk 1]
```diff
@@ -134,6 +132,150 @@ So, for extra small (`xs`) screens for example, the available `.visible-*-*` cla
...
+  }
+}
+@media (min-width: $screen-lg-min) {
+  .nobg-lg {
+    background-image: none !important;
+  }
+}
+
+@media (max-width: $screen-xs-max) {
+  .nobg-le-xs {
```
Comment (stubbornella) [inline src/pivotal-ui/components/utils/utils.scss:259]: what does "le" stand for?

**Comment by aredridel** [comment]
"less than or equal"

**Comment by aredridel** [comment]
Not sure what description to add other than the title — it's a pretty specific class.

**Comment by aredridel** [comment]
screen shot 2016-03-10 at 02 53 44

**Comment by stubbornella** [comment]
I think the le classes are very specific to this particular use case, but ultimately kind of strange for a couple reasons:
1. They don't fit with how bootstrap does responsive utils
2. why wouldn't you have greater than classes if you were going to have less than classes

I also find the class name kind of inscrutable even in the context of it's own file which means it will be far harder to grock in the html file amongst all the other classes. 

I'd suggest we remove the le commit from this, as the other commit does everything we need, is in line with bootstrap, and doesn't raise as many api concerns.

**Comment by aredridel** [comment]
Hm. The -le- ones were actually the ones I intended to use, so I didn't have to do `nobg-xs nobg-sm nobg-md`, but instead `nobg-le-md`. How likely is it we'd add more breakpoints or change these?

**Comment by aredridel** [comment]
And if greater than classes are needed for completeness, sure, but I wouldn't see ever using them since removing information as you go smaller is totally a thing you do but as you get larger ... not so much

**Comment by aredridel** [comment]
The `le` atom came from the IE conditional tags :smiling_imp:

**Comment by aredridel** [comment]
Actually! I just discovered that bootstrap grids _do_ work equivalently: they affect all larger sizes, so you don't have to specify eleventy classes on each thing. This goes the other direction, but for equivalent reasons.

**Pull request closed without merging**